frontend/src/base.html

```diff
@@ -102,29 +102,26 @@ <h2>
   <span>Revision <samp>{{ revision }}</samp> from {{ date }}</span>
 </script>
 
-<header>
-  <div id="menu_browser">
-    <a href="#zero:">View the zero coverage report</a>
-    &bull;
-    <input type="text" id="revision" placeholder="Mercurial revision" </input>
-  </div>
+<script id="menu_browser" type="x-tmpl-mustache">
+  <a href="#view=zero&third_party=on&cpp=on&js=on=java=on&rust=on">View the zero coverage report</a>

```

Comment:
Do we need to specify all options? Could we have `#view=zero` here and let the view use the default? (I'm just afraid we'll have to keep this updated when we add new options otherwise).

Reply:
Done.